generator/cmd/protoc-gen-gclient/testdata/rust/golden/model.rs

```diff
@@ -0,0 +1,498 @@
+#![allow(dead_code)]
+
+use serde::{Deserialize, Serialize};
+
+
+/// A [Secret][google.cloud.secretmanager.v1.Secret] is a logical secret whose
+/// value and versions can be accessed.
+/// 
+/// A [Secret][google.cloud.secretmanager.v1.Secret] is made up of zero or more
+/// [SecretVersions][google.cloud.secretmanager.v1.SecretVersion] that represent
+/// the secret data.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct Secret {
+
+    /// Output only. The resource name of the
+    ///  [Secret][google.cloud.secretmanager.v1.Secret] in the format
+    ///  `projects/*/secrets/*`.
```

Comment:
No action required, but the documentation after the first line is indented two spaces?

Reply:
added todo.